packages/calcite-components/src/components/link/link.scss

```diff
@@ -1,5 +1,18 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-link-text-color: Specifies the text color of the component.
+ * @prop --calcite-link-icon-start-color: Specifies the color of the component's start position icon.
```

Comment:
Do we have requests for each icon to be a different color? If not I think we could reduce this to `--calcite-link-icon-color`

Reply:
Adam suggested this pattern for the button, and I think it makes sense to allow it. I can't think of an example at the moment but it's not that much extra work and best case is that if there does happen to be cases for it that pop up then it's already there. But LMK what you think and maybe we change it for button as well?